network/endpoint.go

```diff
@@ -17,50 +17,55 @@ const (
-	IfName                string
-	HostIfName            string
-	MacAddress            net.HardwareAddr
-	InfraVnetIP           net.IPNet
-	IPAddresses           []net.IPNet
-	Gateways              []net.IP
-	DNS                   DNSInfo
-	Routes                []RouteInfo
-	VlanID                int
-	EnableSnatOnHost      bool
-	EnableInfraVnet       bool
-	EnableMultitenancy    bool
-	NetworkNameSpace      string `json:",omitempty"`
-	ContainerID           string
-	PODName               string `json:",omitempty"`
-	PODNameSpace          string `json:",omitempty"`
-	InfraVnetAddressSpace string `json:",omitempty"`
+	Id                       string
+	HnsId                    string `json:",omitempty"`
+	SandboxKey               string
+	IfName                   string
+	HostIfName               string
+	MacAddress               net.HardwareAddr
+	InfraVnetIP              net.IPNet
+	LocalIP                  string
+	IPAddresses              []net.IPNet
+	Gateways                 []net.IP
+	DNS                      DNSInfo
+	Routes                   []RouteInfo
+	VlanID                   int
+	EnableSnatOnHost         bool
+	EnableInfraVnet          bool
+	EnableMultitenancy       bool
+	AllowInboundFromHostToNC bool
+	AllowInboundFromNCToHost bool
+	NetworkNameSpace         string `json:",omitempty"`
+	ContainerID              string
+	PODName                  string `json:",omitempty"`
+	PODNameSpace             string `json:",omitempty"`
+	InfraVnetAddressSpace    string `json:",omitempty"`
 }
 
 // EndpointInfo contains read-only information about an endpoint.
 type EndpointInfo struct {
-	Id                    string
-	ContainerID           string
-	NetNsPath             string
-	IfName                string
-	SandboxKey            string
-	IfIndex               int
-	MacAddress            net.HardwareAddr
-	DNS                   DNSInfo
-	IPAddresses           []net.IPNet
-	InfraVnetIP           net.IPNet
-	Routes                []RouteInfo
-	Policies              []policy.Policy
-	Gateways              []net.IP
-	EnableSnatOnHost      bool
-	EnableInfraVnet       bool
-	EnableMultiTenancy    bool
-	PODName               string
-	PODNameSpace          string
-	Data                  map[string]interface{}
-	InfraVnetAddressSpace string
+	Id                       string
+	ContainerID              string
+	NetNsPath                string
+	IfName                   string
+	SandboxKey               string
+	IfIndex                  int
+	MacAddress               net.HardwareAddr
+	DNS                      DNSInfo
+	IPAddresses              []net.IPNet
+	InfraVnetIP              net.IPNet
+	Routes                   []RouteInfo
+	Policies                 []policy.Policy
+	Gateways                 []net.IP
+	EnableSnatOnHost         bool
+	EnableInfraVnet          bool
+	EnableMultiTenancy       bool
+	AllowInboundFromHostToNC bool
+	AllowInboundFromNCToHost bool
+	PODName                  string
+	PODNameSpace             string
```

Comment:
PodNamespace

Reply:
i didn't change this..so not changing it